queries/typescript/highlights.scm

```diff
@@ -52,6 +59,9 @@
 (type_parameters
   ["<" ">"] @punctuation.bracket)
 
+(object_type
+  ["{|" "|}"] @punctuation.bracket)
```

Comment:
This looking wrong, I don't remember object types having this syntax

Reply:
it's in the grammar

Reply:
Just found it

Reply:
@lucario387 is right about this. TypeScript doesn't have this syntax.
It is a syntax for [Flow](https://flow.org/en/docs/types/objects/#exact-and-inexact-object-types), and Flow (in Atom) uses the TS grammar.

Reply:
Thanks for the PR. I didn't know that I would need these additions. 👍🏽